components/header.tsx

```diff
@@ -296,33 +308,37 @@ export default function Header(props: { fixed?: boolean }) {
           </button>
         </div>
         <div
-          className={`flex md:items-center p-6 md:p-0 fixed ${
-            mobileMenuOpen ? "right-0" : "right-[-100vw]"
+          className={`flex md:items-center p-6 md:p-0 fixed text-2xl ${
+            mobileMenuOpen ? "right-0" : "right-[-100vw] text-2xl"
           } md:right-auto transition-all md:static ${
             scrolled ? "bg-white/50" : "bg-black/50"
-          } pb-[300px] md:pb-0 md:bg-transparent backdrop-blur-lg md:backdrop-blur-none h-screen md:h-auto top-0 md:top-auto w-screen md:w-auto flex-col md:flex-row text-5xl md:text-base`}
+          } pb-[300px] md:pb-0 md:bg-transparent backdrop-blur-lg md:backdrop-blur-none h-screen md:h-auto top-0 md:top-auto w-screen md:w-auto flex-col md:flex-row md:text-base`}
         >
           <button
             className="md:hidden block absolute top-6 right-8"
             onClick={() => setMobileMenuOpen(false)}
           >
             <XMarkIcon className="h-6 w-6" aria-hidden="true" />
           </button>
-          {navigation.map((item, i) => (
-            <NavigationItemRender
-              item={item}
-              key={i}
-              onHover={(hoverstate, leftOffset) => {
-                if (hoverstate) {
-                  setShowDropdown(i + 1);
-                  triangleRef.current?.style.setProperty(
-                    "left",
-                    `${leftOffset - triangleRef.current.clientWidth / 2}px`
-                  );
-                }
-              }}
-            />
-          ))}
+          <div className={`${mobileMenuOpen ? "h-12" : "flex flex-row" }`}>
```

Comment:
Changes here are possibly causing desktop dropdown to break


https://github.com/user-attachments/assets/07964ce5-b975-4e5a-915e-9f91574c9be6

Reply:
> Changes here are possibly causing desktop dropdown to break
> 
>  Screen.Recording.2024-12-19.at.10.15.00.PM.mov

Yes, you are right. Thank you for pointing out.